com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Configuration.cs

```diff
@@ -439,12 +439,12 @@ void FixHdrpAssetDiffusionProfile(bool fromAsyncUnused)
             if (!IsHdrpAssetUsedCorrect())
                 FixHdrpAssetUsed(fromAsync: false);
 
-            var hdrpAsset = HDRenderPipeline.defaultAsset;
+            var hdrpAsset = HDRenderPipelineGlobalSettings.instance;
```

Comment:
it is not an hdrpAsset anymore. Please rename the variable

Reply:
fixed